tools/webhook-ingestion-pipeline/webhook_endpoint/pubsub_publisher.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+from google.cloud.pubsub_v1 import PublisherClient
+
+
+class PubSubPublisher:
+
+    def __init__(self, project_id):
+        self.project_id = project_id
+        self.client = PublisherClient(
+            client_config = {
+                "interfaces": {
+                    "google.pubsub.v1.Publisher": {
+                        "retry_params": {
+                            "messaging": {
+                                "total_timeout_millis": 650000,
```

Comment:
nit: don't use [magic numbers]
(https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20the%20term,be%20replaced%20with%20named%20constants).
sanity check: this is 650 seconds did you mean 65 seconds? 

prefer: 
```python3
PUBSUB_TIMEOUT = (60 + 5) * 1000  # 1min plus 5s buffer
```

Reply:
changed to const, using 10 min default but as a reference, and cleaned formatting